README.md

```diff
@@ -191,20 +198,39 @@ You may then repeat the --init-token flow to set up a fresh HSM instance.
 
 ## login
 
+A Service Account must be created in the Manetu Realm UI using an x509 certificate prior to login. The certificate is obtained through different means for HSM and ECDSA key based logins.
+
 ### hsm
-You may login using a serial number from the `list` command. Note that the `serial` parameter is 
-optional; if you don't specify, it will pick one from the HSM.
+
+Create the Service Account using the X509 certificate from the [generate](#generate) step.
+
+You may login using a serial number from the [list](#list) command. Note that the `serial` parameter is optional; if you don't specify, it will pick one from the HSM.
 
 ```shell
 $ ./manetu-security-token login hsm --serial 9C:AA:50:2C:B5:1B:01:E2:3D:A6:03:D9:C3:0A:82:6C:F8:8F:6F:D7:B2:E3:CF:05:29:2C:20:F1:AE:C4:7A:72
 ```
 
+The output is a `jwt` that can be used in Manetu API invocation.
+
 ### x509
-You may login using an ECDSA/prime256v1 key with corresponding x509 certificate used to create the manetu service identity.
+
+A sample script to generate the x509 certificate for development purposes:
 
 ```shell
-$ manetu-security-token login x509 --key /path/to/key.pem --cert /path/to/cert.pem --path
+$ openssl ecparam -genkey -name prime256v1 -noout -out key.pem
+$ openssl req -new -x509 -key key.pem -out cert.pem -days 365 -subj "/O=the-realm-of-the-service-account/"
 ```
+
+Replace `the-realm-of-the-service-account` appropriate realm ID.
+
+Log into the realm in Manetu Realm UI and create the Service Account using `cert.pem`.
+
+You may login using the key/cert:
+
+```shell
+$ ./manetu-security-token login x509 --key /path/to/key.pem --cert /path/to/cert.pem --path
```

Comment:
what does --path do here?

Reply:
Without `--path` option, `--key` and `--cert` values will be treated as strings.